.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Can we remove this file? It's the configuration file for VSCode, it shouldn't be checked in.